### PR TITLE
[MNO-686] Fix popover not closing

### PR DESCRIPTION
**Pull request by hedudelgado**

### CODE REVIEW

File: src/app/components/dashboard-apps-dock/no-apps-notification.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="close top-right-corner" type="button" aria-label="Close" ng-click='toggle()'>x</div>
+<div class="close top-right-corner" type="button" aria-label="Close" ng-click='$parent.isPopoverShown = false'>x</div>
```
Comment (cesar-tonnoir) [inline src/app/components/dashboard-apps-dock/no-apps-notification.html]: I think that should work, but @alexnoox would ask you to put that in a method :) Can you do this quick change plz?

Comment (hedudelgado) [reply src/app/components/dashboard-apps-dock/no-apps-notification.html]: Hi @cesar-tonnoir , thans for your comment. Only one quesiton.
 I would be creating a controller only to handle that, Is really worth it to add "so much code" only for this?

Comment (alexnoox) [reply src/app/components/dashboard-apps-dock/no-apps-notification.html]: It would be better to have a method `$parent.closePopover()`

Comment (hedudelgado) [reply src/app/components/dashboard-apps-dock/no-apps-notification.html]: That's a better one! commit pushed @alexnoox

File: src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee [hunk 2]
```diff
@@ -10,6 +10,7 @@ DashboardAppsDockCtrl = ($scope, $cookies, $uibModal, $window, $timeout, MnoeOrg
   $scope.popoverTemplateUrl = 'app/components/dashboard-apps-dock/no-apps-notification.html'
   $scope.isMarketplaceEnabled = MARKETPLACE_CONFIG.enabled
   $scope.isOnboardingEnabled = ONBOARDING_WIZARD_CONFIG.enabled
+  $scope.isPopoverShown = $scope.isOnboardingEnabled && $scope.isMarketplaceEnabled && !$scope.isLoading && _.isEmpty($scope.apps)
```
Comment (alexnoox) [inline src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee]: This should stay a function, `!$scope.isLoading` will only be evaluated at init time.

Comment (hedudelgado) [reply src/app/components/dashboard-apps-dock/dashboard-apps-dock.coffee]: Sorry @alexnoox , I went to add the fix blindly. In that case, can you please have a look at the new commit?

**Comment by alexnoox** [comment]
@hedudelgado Please rebase on `1.1`

**Comment by hedudelgado** [comment]
done @alexnoox 
Thanks

**Pull request merged**